src/backend/src/prisma/manual.ts

```diff
@@ -65,6 +64,32 @@ const activeUserMetrics = async () => {
   // select extract(week from "created") as wk, count(distinct "userId") as "# users", count(distinct "sessionId") as "# sessions" from "Session" group by wk order by wk;
 };
 
+/**
+ * migrate all Change Requests to use Proposed Solutions
+ */
+const migrateToProposedSolutions = async () => {
```

Comment:
Does this cover approved CRs and stuff?

Reply:
Aka does it cover marking which one is approved when the CR is approved and marking all as not approved if the CR is denied?